0.3.6b0

```diff
@@ -0,0 +1,2 @@
+Requirement already satisfied: properties in c:\users\sarah\anaconda3\lib\site-packages
```

Comment:
This file should not have been added, would you mind removing it?

Reply:
Line will be removed

Reply:
correction. File will be removed.